Blockzilla/en.lproj/Localizable.strings

```diff
@@ -344,31 +344,19 @@
 "Settings.sectionSecurity" = "SECURITY";
 
 /* Label for toggle on main screen */
-"Settings.toggleBlockAds" = "Block ad trackers";
```

Comment:
@bogdancernea28 there is no harm to this change to the `en.lproj/Localizable.strings` file but I am curious if you did this manually or if this was some automatic result of changing strings in `UIConstants.swift`.

Asking because I'm focussing on l10n process and tools and I think right now this file will be overwritten during our string import process .. As soon as we import strings, all the `.strings` files are generated / overwritten. At least I think that is what will happen.

Reply:
@st3fan I did it manually.